config/site-config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   primaryColor: "#153cff",
   copyright: "Made by Crisp Studio",
   asideMenu: [
-    { title: "Homepage", link: "https://crisp.studio" },
```

Comment:
I'd put this in a separate PR because it has nothing to do with the venture docs.

Reply:
That's true. I will keep this in mind for next time and just push the change on this branch.